tooling/nargo_fmt/tests/input/constructor.nr

```diff
@@ -0,0 +1,8 @@
+struct Point {
+    x: u8,
+    y: u8,
+}
+
+fn foo() {
+    Point{x :5,y:  10   };
+}
```

Comment:
Can you split this onto multiple lines so it is clearer that the formatter will put it all on one line?